webcentral-backend-videoportal/src/main/java/hu/bme/sch/bss/webcentral/dao/VideoDao.java

```diff
@@ -0,0 +1,18 @@
+package hu.bme.sch.bss.webcentral.dao;
+
+import hu.bme.sch.bss.webcentral.model.Video;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+@Repository
+@Transactional
+public interface VideoDao extends JpaRepository<Video, Long> {
+
```

Comment:
Why don't you use that?
List<Video> findByVisible(boolean isVisible)

Reply:
At this moment we only need querry with visibly=true and properbly won't user other way, if yes probably a more complex filtering will be needes, so it'll be refactored